[contributing doc] explain/link to good first issue (#11346)
huggingface/transformers: CONTRIBUTING.md

* Contributing to the examples or to the documentation;
* Submitting issues related to bugs or desired new features.

In particular there is a special [Good First
Issue](https://github.com/huggingface/transformers/contribute) listing. Tt will give you a list of
open Issues that are open to anybody to work on. Just comment in the issue that you'd like to work
on it. In that same listing you will also find some Issues with `Good Second Issue` label. These are
typically slightly more complicated than the Issues with just `Good First Issue` label. But if you

> feel you know what you're doing go for it.

Reviewer: ```suggestion
feel you know what you're doing, go for it.
```

### Did you find a bug?

> The `transformers` are robust and reliable thanks to the users who notify us of

Reviewer: If we format in code, it becomes the library IMO, so it should be
```suggestion
The 🤗 Transformers library is robust and reliable thanks to the users who notify us of
```
